server/src/test/java/org/elasticsearch/action/search/CanMatchPreFilterSearchPhaseTests.java

```diff
@@ -133,6 +133,67 @@ public void run() throws IOException {
         }
     }
 
+    public void testFilterWithRollup() throws InterruptedException {
```

Comment:
I am not sure what is tested here ? I don't see where the logic to replace an index is tested ?

Reply:
This test was accidentally left out. All tests have been implemented in `RollupSearchIT`